.github/workflows/01-build-and-release.yml

```diff
@@ -4,28 +4,21 @@ on:
   push:
     branches:
       - main
-      - set-up-cicd
+      - use-preinstalled-djgpp-image-in-workflows
```

Comment:
This line should be removed before merging this PR.

Reply:
This workflow could also be run for all PRs against `main`, but skip the Github releases part (using [conditions](https://docs.github.com/en/actions/using-jobs/using-conditions-to-control-job-execution)) if we're not on `main`?